<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbAdaptDialogBase</class>
 <widget class="QDialog" name="dbAdaptDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>275</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Datenbank aktualisieren</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Anpassung starten ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>Anpassung starten ...</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_LayersAdapt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>420</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Zu aktualisierende Datenbank</string>
   </property>
   <widget class="QPushButton" name="pb_selectQKanDB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Suchfenster öffnen ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Suchfenster öffnen ...</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tf_qkanDB">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aktuelle QKan - Datenbank&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Aktuelle QKan - Datenbank</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_updateQkanDB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>420</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>vor Schließen der Projektdatei ...</string>
   </property>
   <widget class="QRadioButton" name="rbSave">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Soll das Projekt vor der Bearbeitung gespeichert werden?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Projekt vor der Bearbeitung speichern</string>
    </property>
    <property name="text">
     <string>Änderungen speichern</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDiscard">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Soll das Projekt vor der Bearbeitung verworfen werden?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Projekt vor der Bearbeitung verwerfen</string>
    </property>
    <property name="text">
     <string>Änderungen verwerfenn</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>dbAdaptDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>dbAdaptDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
